src/solvers/ODE.jl

```diff
@@ -47,61 +47,103 @@ function ODEjl_wrapper(tr::TestRun)
     return tend, yend, stats
 end
 
-#@require ODE begin # bug!
 import ODE
-begin
-    ode_only = 0 # dae index
-    pkg = "ODE.jl"
+
+
+##############################################################################
+#List of all ODE.jl solvers avaible for testing
+##############################################################################
+ODE_release =true
+ODE_pwl = !ODE_release
+if ODE_release
+    ## Non-stiff fixed step solvers
+    nonstiff_fixedstep= [
+               ODE.ode1,
+               ODE.ode2_midpoint,
+               ODE.ode2_heun,
+               ODE.ode4,
+               ODE.ode4ms,
+               ODE.ode5ms
+    #          ODE.ode_imp_ab #Implicit Adam Bashforth under construction
+               ]
+    ## Non-stiff adaptive step solvers
+    nonstiff_adaptive=[
+    #          ODE.ode21, # this fails on Travis with 0.4?! TODO revert once fixed.
+               ODE.ode23,
+               ODE.ode45,
+               ODE.ode45_dp,
+               ODE.ode45_fe,
+               ODE.ode78,
+               ODE.ode_ab_adaptive
+               ]
+    # Stiff fixed-step solvers
+    stiff_fixedstep=[
+               ODE.ode4s_s,
+               ODE.ode4s_kr
+               ]
+    #Stiff adaptive solvers
+    stiff_adaptive = [
+               ODE.ode23s
+     #          ODE.odeRadauIIA #RADAU methods under construction
+               ]
+elseif ODE_pwl
```

Comment:
Let's get rid of this pwl, stuff.  You can put that into the file which you use to run his PR.  Just do `delete!` for the solvers which don't work.